core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -171,7 +171,8 @@ export class LoggedOutUser extends BaseUser {
 
     expect(this.page.url())
       .withContext(
-        `${buttonName} should open the ${expectedDestinationPageName} page`
+        `${buttonName} should open the ${expectedDestinationPageName} page,
+          but it opens ${this.page.url()} instead.`
```

Comment:
Can you also add this printing this.page.url() part to the rest? We want to make our code consistent.

Reply:
Reverting to the erstwhile one. As expect().toBe() prints a clear message itself. 

However, have added consoling of this.page.url() elsewhere we are not using expect().toBe().